src/implementations/react/src/elements/components/GlobalNav/TopNav/Help/Group.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import GroupAdapter from '../../../../../adapters/GlobalNav/TopNav/Help/GroupAdapter';
+
+class Group extends Component {
+  static propTypes = {
+  }
+
+
+  render() {
+    const { ...otherProps } = this.props;
+    return (<GroupAdapter {...otherProps} />);
```

Comment:
This file actually does nothing. Use the GroupAdapter in place of this and delete this file.